src/styles/blocks/cards/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $card-bg-color;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  transition: all 300ms;

```

Comment:
Don't use `all` for transition, it affects in bad way on the website performance
Mention only needed properties